scripts/apps/authoring/authoring/directives/ArticleEditDirective.js

```diff
@@ -319,6 +321,40 @@ export function ArticleEditDirective(
                 });
             };
 
+            /**
+             * Shows the preview for the given embed field.
+             */
+            scope.previewEmbed = (field) => {
+                document.getElementById('embed-preview-' + field._id).innerHTML = scope.item.extra[field._id];
+            };
+
+            /**
+             * Hides the preview for the given embed field.
+             */
+            scope.hideEmbedPreview = (field) => {
+                document.getElementById('embed-preview-' + field._id).innerHTML = null;
```

Comment:
Use angular scope to set content inside the template, that's why we are using Angular... We don't have to directly overwrite `innerHTML`.